entityx/help/Pool_test.cc

```diff
@@ -64,6 +64,8 @@ TEST_CASE("TestPoolPointers") {
   REQUIRE(extrapolated_p16 !=  static_cast<void*>(p16));
   void *extrapolated_p24 = p16 + 8 * sizeof(Position);
   REQUIRE(extrapolated_p24 !=  static_cast<void*>(p24));
+
+  for (auto ptr : ptrs) delete[] ptr;
```

Comment:
Why not use unique_ptr instead?

Reply:
No real reason, I figured it might be best to leave the tests relatively untouched so I just added the cleanup. I can definitely just use unique pointers instead though which should be cleaner.